features/flows/elastic/log4j2.properties

```diff
@@ -0,0 +1,74 @@
+status = error
```

Comment:
I'm guessing this doesn't belong here.

Reply:
I am not sure. I just copied over the official elastic `log4j2.properties` file and modified it. I kept this line. So I would keep it for now. 

Source: https://github.com/elastic/elasticsearch/blob/master/distribution/src/main/resources/config/log4j2.properties